core/scss/components/activity-button/tests/activity-button.cy.js

```diff
@@ -3,6 +3,7 @@ describe('@collab-ui/core', function() {
     cy.visit(`${Cypress.env('BASE_URL')}/activity-button`)
       .get('.cui-button__container--small')
       .should('be.visible')
+      .wait(500)
```

Comment:
Feels like we can have a util function for this repeated code
```js
function percySnapshotForVisibleElement(path, target) {
    cy.visit(`${Cypress.env('BASE_URL')}/${path}`)
      .get(target)
      .should('be.visible')
      .wait(500)
      .percySnapshot();
}
```

so that it can be used elsewhere as 
```js
percySnapshotForVisibleElement('/activity-button', '.cui-button__container--small')
```